shell/browser/hid/electron_hid_delegate.cc

```diff
@@ -117,14 +127,18 @@ const device::mojom::HidDeviceInfo* ElectronHidDelegate::GetDeviceInfo(
     content::BrowserContext* browser_context,
     const std::string& guid) {
   auto* chooser_context = GetChooserContext(browser_context);
+  if (!chooser_context)
+    return nullptr;
```

Comment:
This matches upstream, but if we keep it as is we need to change `GetChooserContext` to return nullptr if !browser_context.